common/db.go

```diff
@@ -144,8 +144,7 @@ func WaitDBUp() {
 	defer func() {
 		sdb.Close()
 	}()
-	for _, err := dtmimp.DBExec(sdb, "select 1"); err != nil; { // wait for mysql to start
```

Comment:
When using docker-compose up, dtm is started before mysql sometimes. So I add WaitDBUp for dtm to wait for db up instead of just exit.

If password is error, then the output hint is clear for user to take action. So that I do not distinguish the connection refuse and other error

Reply:
Get